site/config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = ""
+baseurl = "ocw-next.netlify.com"
```

Comment:
This looks great locally.  On the deploy preview if you hit the placeholder landing page, you'll notice that the link to courses is rendering as https://deploy-preview-99--ocw-next.netlify.com/ocw-next.netlify.com/courses/.  Looking at the placeholder index, the link is like this:

```
<a href="./courses/">
  course listing
</a>
```

I think if we set the base URL, we're going to have to use the absURL or relURL shortcodes contextually for any link in any template contextually.

Reply:
ugh what a mess :woman_facepalming:

Reply:
Yea, luckily crawling through the code I think that's actually the only raw relative URL hard coded in our templates, so you should be good to just change that one.